README.md

```diff
@@ -16,46 +16,48 @@
 [Basic Usage](#basic-usage) |
 [Contributing](#contributing)
 
-
 ## Key Features
+
 Stumpless offers a robust set of features to make logging in C faster and
 easier:
- * easy logging to [lots of things](#what-can-it-log-to) like Splunk, rsyslog,
-   journald, the Windows Event Log, and more!
- * structured and unstructured logging to suit your needs
- * interoperable with common log daemons and libraries
- * builds for Linux, Windows, Mac, MinGW, MSYS2, Cygwin, and more
- * completely thread safe
- * can be adjusted or removed during compilation for zero runtime impact
- * localized for multiple languages :brazil: :bulgaria: :cn: :czech_republic:
-   :de: :es: :fr: :greece: :india: :it: :poland: :slovakia: :sweden: :us: :denmark: :israel:
-   ([add yours!](https://github.com/goatshriek/stumpless/blob/latest/docs/localization.md))
- * easy-access
-   [documentation](https://goatshriek.github.io/stumpless/docs/c/latest/index.html),
-   [examples](https://github.com/goatshriek/stumpless/tree/latest/docs/examples),
-   and [support](https://gitter.im/stumpless/community).
 
+- easy logging to [lots of things](#what-can-it-log-to) like Splunk, rsyslog,
+  journald, the Windows Event Log, and more!
+- structured and unstructured logging to suit your needs
+- interoperable with common log daemons and libraries
+- builds for Linux, Windows, Mac, MinGW, MSYS2, Cygwin, and more
+- completely thread safe
+- can be adjusted or removed during compilation for zero runtime impact
+- localized for multiple languages :brazil: :bulgaria: :cn: :czech_republic:
+  :de: :es: :fr: :greece: :india: :it: :poland: :slovakia: :sweden: :us: :denmark: :israel: :kenya:
```

Comment:
This line is the right place for the 🇰🇪 character, however the list bullet points seem to have been inadvertently changed. The only change that should be included in this file is addition of the flag - please revert the other changes.